### PR TITLE
Feat/terabee

### PR DESCRIPTION
**Pull request by chrisandoX**
Fixed, incorrect device profile settings. Added 2 new products and updated the Terabee commercial information.

### CODE REVIEW

File: vendors/terabee/models/pc-l-xl/model.yaml
Comment (tomHallActility) [inline vendors/terabee/models/pc-l-xl/model.yaml:1]: Unmatched example description

File: vendors/terabee/models/tb-pcm/model.yaml
Comment (tomHallActility) [inline vendors/terabee/models/tb-pcm/model.yaml:1]: Unmatched example description

File: vendors/terabee/models/tb-poc/model.yaml
Comment (tomHallActility) [inline vendors/terabee/models/tb-poc/model.yaml:1]: Unmatched example description

**Review by tomHallActility — CHANGES_REQUESTED** [review]
Hello Terabee,

The examples field in the model.yaml files must be filled with the right protocolId (which is the case) and have the descriptions of the examples (of that driver) you want to associate with your models.
Could you change that? It's the only critical thing I cannot change myself.
Thank you very much.

Best regards,
Tom

**Comment by chrisandoX** [comment]
Requested changes have been adressed. I've also noticed a device profiles that weren't used hence removed it for clarity.

**Review by tomHallActility — APPROVED** [review]
Thank you for your changes.
I'll merge your branch.
I will still need to change a few necessary technical things myself before we can deploy it.

**Comment by chrisandoX** [comment]
Thanks!

When do you think it would be ready for deployment, I am looking forward to
test it ;)

BR
Chris

**Comment by tomHallActility** [comment]
No problem at all.
We're planning on deploying this afternoon.

Best regards,
Tom

**Pull request merged**